build-tools/packages/build-cli/docs/generate.md

```diff
@@ -246,6 +247,57 @@ DESCRIPTION
 
 _See code: [src/commands/generate/entrypoints.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/generate/entrypoints.ts)_
 
+## `flub generate typetests`
+
+Generates type tests for a package or group of packages.
+
+```
+USAGE
+  $ flub generate typetests [-v | --quiet] [--level public|alpha|beta|internal|legacy] [--publicFallback] [--concurrency
+    <value>] [--all | --dir <value> | --packages | -g client|server|azure|build-tools|gitrest|historian|all |
+    --releaseGroupRoot client|server|azure|build-tools|gitrest|historian|all] [--private] [--scope <value> | --skipScope
+    <value>]
+
+FLAGS
+  --concurrency=<value>  [default: 25] The number of tasks to execute concurrently.
+  --level=<option>       [default: internal] What API level to generate tests for.
```

Comment:
This level is used for both current and previous, so we're comparing the same level API between versions. However, because of the public fallback logic, one of them could be a different API level than the other. I think this is the desired behavior.

In the future if it's useful to create tests between versions AND level, then we may add more flags.

Reply:
The (silent) fallback logic feels like potentially very surprising behavior for someone not familiar with the command, I'm not a fan of it as default. I'd rather have the tool throw if either version does not have the specified level and the error message can suggest "pass --publicFallback to enable fallback logic". Even if we have to call it with the flag everywhere because that's the behavior we want, I'd still prefer to see it explicilty.

Reply:
Fair enough. I'll update the defaults. Since we've already shipped api trimming we may not need the fallback anyway, and we can enable it explicitly if needed.